docs/installation/bootstrap.rst

```diff
@@ -62,6 +62,10 @@ Configuration
         minion: <hostname-of-the-bootstrap-node>
       archives:
         - <path-to-metalk8s-iso>
+      kubernetes:
+        apiserver:
+          featuresGates:
```

Comment:
This should be `featureGates` (also in the implementation, if applicable).

Reply:
--' MB again, I did it too fast :smiley: